unittests/DependencyScan/Features.cpp

```diff
@@ -64,6 +64,6 @@ TEST_F(ScanTest, TestDoesNotHaveArgumentQuery) {
   bool hasOption;
   hasOption = optionSet.find("-clearly-not-a-compiler-flag") != optionSet.end();
   EXPECT_EQ(hasOption, false);
-  hasOption = optionSet.find("-emit-modul") != optionSet.end();
+  hasOption = optionSet.find("-emit-module") != optionSet.end();
```

Comment:
I believe that this is intentionally verifying that `optionSet.find()` does not prefix match on a fragment of an option that is present.